front/src/components/atoms/HamburgerMenu.tsx

```diff
@@ -9,11 +9,10 @@ type HamburgerMenuProps = {
 
 const StyledBurger = styled.button<HamburgerMenuProps>`
   position: absolute;
-  left: 0;
+  left: -15px;
```

Comment:
I think there is a "cleaner" way of doing this instead of using position absolute, and setting hard coded left value.
Since this button is inside a flex container with "justify-end" you could set margin-right "auto" on this button and this will push it all the way to the left.
Then simply add more padding left/right on the Navbar component.

Reply:
Thanks for your comment. I didn't know how to change the position of that component. Now I have seen that there was a "margin-left" that was pushing the "Hamburger" to the right. This way it is much cleaner. Thank you for the correction